tests/e2e/data/test_data_cp.py

```diff
@@ -0,0 +1,200 @@
+import logging
+import re
+import sys
+from tempfile import TemporaryDirectory
+from typing import Iterable, Iterator, List, Optional
+
+import pytest
+from pytest_lazyfixture import lazy_fixture  # type: ignore
+from tenacity import retry, stop_after_attempt, stop_after_delay
+
+from .cloud_to_local import generate_cloud_to_local_copy_configs
+from .cloud_to_platform import generate_cloud_to_platform_copy_configs
+from .local_to_cloud import generate_local_to_cloud_copy_configs
+from .local_to_local import generate_local_to_local_copy_configs
+from .platform_to_cloud import generate_platform_to_cloud_copy_configs
+from .resources import CopyTestConfig, _wrap_test_config_into_yield_fixture
+from .utils import _resume_fixture_iterator, _run_command
+
+
+UUID4_PATTERN = r"[0-9a-f]{8}-[0-9a-f]{4}-[4][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}"
+DISK_ID_PATTERN = rf"disk-{UUID4_PATTERN}"
+DISK_ID_REGEX = re.compile(DISK_ID_PATTERN)
+
+root_logger = logging.getLogger()
+root_logger.setLevel(logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+TEST_LOCAL_TO_LOCAL = True
+TEST_CLOUD_TO_LOCAL = True
+TEST_LOCAL_TO_CLOUD = True
```

Comment:
The functionality, which is tested in `TEST_CLOUD_TO_LOCAL` and `TEST_LOCAL_TO_CLOUD` is covered also in `TEST_CLOUD_TO_PLATFORM` and `TEST_PLATFORM_TO_CLOUD`, isn't it?

Suggestion: run those tests once, ensure they pass, and disable them until some related changes appear. (we probably should put those boolean control variables config into `conftest.py`)

End goal: decrease the overall time of running the tests and their cost since most of the time we are getting only `pip` updates PRs. WDYT?

Reply:
Indeed, under the hood cloud-to-local and local-to-cloud are being tested indirectly during cloud-to-platform and platform-to-cloud.